docker/jenkins-network-six-nodes.yml

```diff
@@ -4,47 +4,47 @@ services:
     environment:
       JAVA_OPTS: -server -Xms2g -Xmx2g -Dcom.sun.management.jmxremote.port=9011 -Dcom.sun.management.jmxremote.rmi.port=9011 -Dcom.sun.management.jmxremote.authenticate=false -Dcom.sun.management.jmxremote.ssl=false -Djava.rmi.server.hostname=localhost -agentlib:jdwp=transport=dt_socket,address=50505,suspend=n,server=y
     build: .
-    image: radixdlt.azurecr.io/radixdlt/radixdlt-core:develop
+    image: radixdlt/base-core:latest
```

Comment:
This is radix core which is built on top of base core . Image name should be 
`image: radixdlt/radixdlt-core:develop
`
Image here refers to one that gets built when docker-compose gets run.

Reply:
OK, change looks good.